package/yast2-network.changes

```diff
@@ -3,6 +3,8 @@ Thu Jul 12 11:46:53 UTC 2018 - mfilka@suse.com
 
 - bnc#964856
   - dropped calls to obsolete NetworkInterfaces API
+  - mentioning device type explicitly using new option is mandatory
+    since now when adding device using CLI
```

Comment:
I would rephrase it a bit: "CLI: for new devices the option type is now mandatory"